pkg/crosscluster/replicationtestutils/testutils.go

```diff
@@ -186,19 +187,18 @@ func (c *TenantStreamingClusters) init(ctx context.Context) {
 func (c *TenantStreamingClusters) StartDestTenant(
 	ctx context.Context, withTestingKnobs *base.TestingKnobs, server int,
 ) func() {
+	var err error
+	var testKnobs base.TestingKnobs
```

Comment:
just to confirm this change was necessary to get access to the destination tenant's jobregistry, right?

Reply:
Yup